lib/helpers/replace.js

```diff
@@ -0,0 +1,8 @@
+function convertible(value) {
+	if (value === undefined) return "";
+	if (value === null) return "";
+	if (typeof value.toString === "function") return value;
+	return typeof value;
```

Comment:
Why return the typeof and not the value?

Reply:
Because value doesn't implement `toString` method and can't be used by `String.prototype.replace`

Reply:
for reproduce:
```js
const o1 = { a: 5 }
const o2 = { toString: 5 }
const s = "abc"
s.replace("b", o1) // worked
s.replace("b", o2) // fail

```